### PR TITLE
add more configuration parameters

### PR DESCRIPTION
**Pull request by Trisfald**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,11 +17,16 @@ To run a single benchmark, run: `./run_single_bench.sh <BENCHMARK_NAME>` .
...
-The maximum number of cpus used by server is governed by the environmental variable `GRPC_SERVER_CPUS`. Defaults to `1`.
-
-Which tests to run is governed by a regex in `GRPC_BENCHMARK_SELECT`, for example `rust` or `(kotlin|java)`. Defaults to all tests.
+The benchmark can be configure through the following environment variables:
+
+|**Name**|**Description**|**Default value**|
+|--------|---------------|:---------------:|
+|GRPC_BENCHMARK_DURATION|Duration of the benchmark.|30s|
+|GRPC_SERVER_CPUS|Maximum number of cpus used by the server.|1|
+|GRPC_BENCHMARK_SELECT|Which tests to run. For example `rust` or `(kotlin|java)`.|. (*all tests*)|
```
Comment (LesnyRumcajs) [inline README.md:26]: I think the formatting fiascoed on this line, looking at the `rich` diff

Comment (berestovskyy) [inline README.md:26]: There is no more GRPC_BENCHMARK_SELECT on master after my change. Now we can either specify the list of benchmarks to run: `./bench.sh bench1 bench2...` or use shell patterns, i.e. `./bench.sh java* rust* cpp*`

Comment (Trisfald) [reply README.md:26]: Haha indeed

Comment (Trisfald) [reply README.md:26]: Yeah I'll need to do some good old `rebasing`

File: README.md [hunk 2]
```diff
@@ -15,9 +15,15 @@ To run the benchmarks use: `./bench.sh [BENCH1] [BENCH2] ...` . They will be run
 To clean-up the benchmark images use: `./clean.sh [BENCH1] [BENCH2] ...`
 
 ## Configuring the benchmark
-Benchmark duration is governed by environmental variable `GRPC_BENCHMARK_DURATION`. Defaults to `30s`.
-
-The maximum number of cpus used by server is governed by the environmental variable `GRPC_SERVER_CPUS`. Defaults to `1`.
+The benchmark can be configure through the following environment variables:
```
Comment (LesnyRumcajs) [inline README.md:18]: configure**d**

Comment (Trisfald) [reply README.md]: done

**Pull request merged**